src/app/schemas/gene.ts

```diff
@@ -1,67 +1,7 @@
 import { Schema, Model, model } from 'mongoose';
 import { GeneDocument } from '../models';
 
-export let GeneSchema: Schema = new Schema(
-    {
-        ensembl_gene_id: {
-            required: true,
-            type: String,
-        },
-        hgnc_symbol: {
-            required: true,
-            type: String,
-        },
-        logfc: {
-            required: false,
-            type: Number
-        },
-        fc: {
-            required: false,
-            type: Number
-        },
-        ci_l: {
-            required: false,
-            type: Number
-        },
-        ci_r: {
-            required: false,
-            type: Number
-        },
-        adj_p_val: {
-            required: false,
-            type: Number
-        },
-        tissue: {
-            required: false,
-            type: String
-        },
-        study: {
-            required: false,
-            type: String
-        },
-        model: {
-            required: false,
-            type: String
-        }
-    }, {
-        timestamps: true
-    }
-);
-
-// Add methods here, if needed e.g.
-/*GeneSchema.methods.fullName = () => {
-    return (this.firstName.trim() + " " + this.lastName.trim());
-};*/
-
-// Handler **must** take 3 parameters: the error that occurred, the document
-// in question, and the `next()` function
-GeneSchema.post('save', function(error, doc, next) {
-    if (error.name === 'MongoError' && error.code === 11000) {
-        next(new Error('There was a duplicate key error'));
-    } else {
-        next(error);
-    }
-});
+export let GeneSchema: Schema = new Schema({}, { collection: 'genes' });
```

Comment:
I'm guessing the above code was unnecessary + duplicated from models?

Reply:
Ditto for all the other similar changes in the schema folder

Reply:
It turns out that mongoose schemas are only required to specify write behaviors (https://stackoverflow.com/questions/28630865/mongoose-read-only-without-schema), and since we don't actually ever write to the database all of that code was unnecessary and confusing since some of the schemas were out of sync with the actual data in the db.